### PR TITLE
Lightbox and Screenfull: Tweak - These utilities were loaded by default instead of only when needed - ED-448

### PR DESCRIPTION
**Pull request by rotemee**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/frontend/handlers/video.js [hunk 1]
```diff
@@ -63,12 +59,28 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 		}
 	}
 
-	animateVideo() {
-		this.getLightBox().setEntranceAnimation( this.getCurrentDeviceSetting( 'lightbox_content_animation' ) );
+	async getLightbox() {
```
Comment (kobizz) [inline assets/dev/js/frontend/handlers/video.js:62]: Remove this method

File: assets/dev/js/frontend/utils/lightbox/lightbox-manager.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		event.preventDefault();
+
+		if ( editMode && ! elementor.getPreferences( 'lightbox_in_editor' ) ) {
+			return;
+		}
+
+		this.showLightbox( element );
+	}
+
+	async showLightbox( element ) {
```
Comment (kobizz) [inline assets/dev/js/frontend/utils/lightbox/lightbox-manager.js:61]: Try moving it to the lightbox itself

File: assets/dev/js/frontend/utils/url-actions.js [hunk 3]
```diff
@@ -13,11 +13,15 @@ export default class extends elementorModules.ViewModule {
 
 	initActions() {
 		this.actions = {
-			lightbox: ( settings ) => {
+			lightbox: async ( settings ) => {
+				const lightbox = elementorFrontend.config.experimentalFeatures.e_optimized_assets_loading ?
```
Comment (kobizz) [inline assets/dev/js/frontend/utils/url-actions.js:17]: Always use `await`

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/preloaded-modules.js [hunk 4]
```diff
@@ -20,4 +22,11 @@ elementorFrontend.elements.$window.on( 'elementor/frontend/init', () => {
 		'image-carousel.default': ImageCarousel,
 		'text-editor.default': TextEditor,
 	};
+
+	elementorFrontend.on( 'components:init', () => {
+		// We first need to delete the property because by default it's a getter function that cannot be overwritten.
+		delete elementorFrontend.utils.lightbox;
+
+		elementorFrontend.utils.lightbox = new LightboxModule()
```
Comment (kobizz) [inline assets/dev/js/frontend/preloaded-modules.js:30]:
```suggestion
		elementorFrontend.utils.lightbox = new LightboxModule();
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/handlers/video.js (hunk 1 above)
Comment (rotemee) [reply assets/dev/js/frontend/handlers/video.js:62]: Done.

File: assets/dev/js/frontend/utils/lightbox/lightbox-manager.js (hunk 2 above)
Comment (rotemee) [reply assets/dev/js/frontend/utils/lightbox/lightbox-manager.js:61]: Done.

File: assets/dev/js/frontend/utils/url-actions.js (hunk 3 above)
Comment (rotemee) [reply assets/dev/js/frontend/utils/url-actions.js:17]: Done.

File: assets/dev/js/frontend/preloaded-modules.js (hunk 4 above)
Comment (rotemee) [reply assets/dev/js/frontend/preloaded-modules.js:30]: Done.

**Review by kobizz — APPROVED** [review]

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 5]
```diff
@@ -169,6 +169,41 @@ module.exports = elementorModules.ViewModule.extend( {
...
+				description: element.dataset.elementorLightboxDescription,
+				modalOptions: {
+					id: 'elementor-lightbox-slideshow-' + slideshowID,
+				},
+			} );
+
+			return;
+		}
+
+		const initialSlideURL = element.dataset.elementorLightboxVideo ? element.dataset.elementorLightboxVideo : element.href;
```
Comment (iNewLegend) [inline assets/dev/js/frontend/utils/lightbox/lightbox.js:202]:
```suggestion
		const initialSlideURL = element.dataset.elementorLightboxVideo || element.href;
```

**Review by iNewLegend — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/utils/lightbox/lightbox.js (hunk 5 above)
Comment (rotemee) [reply assets/dev/js/frontend/utils/lightbox/lightbox.js]: Fixed.

**Review by kobizz — APPROVED** [review]

**Pull request merged**